lib/libuutil/uu_pname.c

```diff
@@ -88,11 +88,17 @@ uu_warn_internal(int err, const char *format, va_list alist)
 	if (pname != NULL)
 		(void) fprintf(stderr, "%s: ", pname);
 
-	if (format != NULL)
+	if (format != NULL) {
 		(void) vfprintf(stderr, format, alist);
 
-	if (strrchr(format, '\n') == NULL)
```

Comment:
This function is quite weird, it only print strerror if format doesn't have '\n'?
Never the less, why not just change this to `if (!format || strrchr(format, '\n') == NULL)`?
That way we would also print strerror when format is NULL.

Reply:
It appears to be entirely dead code as well.  We should simply remove `uu_pname.[ch]` and all of functions it provides.  None of them are used.  It wouldn't hurt to make a pass over the other `uu_` sources and see how much is really needed.

Reply:
I had not realized that. I will make another pass over this to delete as much as I can. Deleting dead code is fun. :)

Reply:
I just pushed a revision that deletes all of this.